genon/preprocessor/tests/unit/test_attachment_textloader_long_line_333.py

```
@@ -0,0 +1,115 @@
+    except ModuleNotFoundError:
+        # 실행 루트에 따라 sys.path 보정 (기존 테스트와 동일 패턴)
+        sys.path.append(str(Path(__file__).resolve().parents[3]))
+        from facade.attachment_processor import _get_pdf_path, TextLoader
+        return _get_pdf_path, TextLoader
+
+
+def _has_weasyprint() -> bool:
+    try:
+        import weasyprint  # noqa: F401
+        return True
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)